packages/web-components/fast-components/src/tooltip/tooltip.styles.ts

```diff
@@ -0,0 +1,94 @@
+import { css } from "@microsoft/fast-element";
+import { forcedColorsStylesheetBehavior } from "@microsoft/fast-foundation";
+import {
+    accentFillActiveBehavior,
+    accentFillHoverBehavior,
+    accentFillRestBehavior,
+    neutralFillHoverBehavior,
+    neutralFillInputActiveBehavior,
+    neutralFillInputHoverBehavior,
+    neutralFillInputRestBehavior,
+    neutralFillRestBehavior,
+    neutralFocusBehavior,
+    neutralForegroundRestBehavior,
+    neutralOutlineRestBehavior,
+} from "../styles/index";
+
+export const TooltipStyles = css`
+    :host {
+        contain: layout;
+        overflow: visible;
+        height: 0;
+        width: 0;
+    }
+
+    .tooltip {
+        box-sizing: border-box;
+        border-radius: calc(var(--corner-radius) * 1px);
+        border: calc(var(--outline-width) * 1px) solid ${accentFillActiveBehavior.var};
+        border-color: ${neutralFocusBehavior.var};
+        box-shadow: 0 0 0 1px ${neutralFocusBehavior.var} inset;
+        background: ${neutralFillRestBehavior.var};
+        color: ${neutralForegroundRestBehavior.var};
+        padding: 4px;
+        height: fit-content;
+        width: fit-content;
+        font: inherit;
+        font-size: var(--type-ramp-base-font-size);
+        line-height: var(--type-ramp-base-line-height);
+        white-space: nowrap;
+        z-index: 10000;
```

Comment:
I see the 10000, I wonder if we should create a semantic ramp for values that we can invoke. This also may be beneficial as a var with a default value. That would make it easy to manage as the owner of an app - especially if CSS Parts are not supported in some of my primary browsers.

Reply:
I think we should have a central way to manage the zindex for modals, tooltips, select popups, etc. For example, what if we have a modal with a select in it and there's a tooltip on something in the select popup? Do we alway treat tooltips as highest, then popups, then modals? Would something simple like that work? Can we just encode that in a simple lookup in foundation that can be used by various components and design systems?

Reply:
Yeah, there are a few ways that we've done this before - I'm happy to open an issue and propose something.

Reply:
I think it's fine for now - we don't have to block on this. Could you add a note w/ a TODO so I can find it :)